bin/afppasswd/afppasswd.c

```diff
@@ -35,19 +35,13 @@
 #include <sys/types.h>
 #include <unistd.h>
 
+#include <gcrypt.h>
+#define DES_KEY_SZ 8
```

Comment:
Do you want to put this in a `#ifndef` block just in case this gets defined in a header somewhere in the future?

I also tried to find out if libgcrypt had an equivalent to `DES_cblock` constant that we could use, but couldn't find traces of one. From my understanding DES block size is always 8 anyways so it's a minor concern.

Reply:
Done.

Reply:
Cheers.